clang/test/SemaHLSL/BuiltIns/clamp-errors.hlsl

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm-only -disable-llvm-passes -verify -verify-ignore-unexpected
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm-only -disable-llvm-passes -verify -verify-ignore-unexpected=note
```

Comment:
would it be ok if we removed the `-verify-ignore-unexpected=note` and just add  the notes?

Reply:
There were a substatial number of notes; this file original had a -verify-ignore-unexpected, so this is no worse than before and I think is actual more accurate to what we don't care about; because no errors or warnings should be ignored.  There are 41 ignored notes, which are almost all
"candidate function not viable: requires 3 arguments, but 1 was provided".  I don't think including them in this test file would provide value.